### ISSUE TITLE
Launch

### ISSUE THREAD

**Issue by RichardLitt** [RichardLitt/egoless-web#8]
We should launch this as an actual extension in the Chrome store!

**Comment by hackergrrl** [comment]
Very much yes!

**Comment by hackergrrl** [comment]
https://developer.chrome.com/webstore/publish

**Comment by hackergrrl** [comment]
We'll also need an icon for the Chrome Store.

**Comment by RichardLitt** [comment]
Logo: How about a cropped version of this? https://pixabay.com/en/person-forest-lush-green-moss-1030884/

**Comment by hackergrrl** [comment]
Cool! Can you say more about your thinking / how it connects for you to the project?

**Comment by RichardLitt** [comment]
Yeah; for me, green, forested areas are where I feel rejuvenated. And I like that the logo would be natural; detailed, irreplaceable, reflective of what we would see in the ancestral environment as opposed to the pared down and idealized designs of vectors.

I also think a lot of time is normally spent on logos, and I think that that time could be better spent elsewhere, but that's not really a productive comment. I'd just rather show a nice picture, and have that be identified with the product, than spending a lot of time trying to find the best combination of themes and symbols to represent it. 

Another option (photo source: me): 

![img_0931](https://cloud.githubusercontent.com/assets/910753/21480138/4a8023e0-cb28-11e6-83e9-5cc88b1bdb1e.jpg)
